metals/src/main/scala/scala/meta/metals/sbtserver/SbtServer.scala

```diff
@@ -48,11 +48,13 @@ object SbtServer extends LazyLogging {
   private def fail(message: String) = Task.now(Left(message))
 
   def readVersion(cwd: AbsolutePath): Option[String] = {
-    val path = cwd.resolve("project").resolve("build.properties")
-    val input = Files.newInputStream(path.toNIO)
     val props = new Properties()
-    Try { props.load(input) }
-    input.close()
+    val path = cwd.resolve("project").resolve("build.properties")
+    if (path.toFile.exists) {
```

Comment:
You can simply do `path.isFile`.

`isFile` uses [`Files.isRegularFile`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#isRegularFile(java.nio.file.Path,%20java.nio.file.LinkOption...)) to determine whether the file is a regular file and that it exists.

Reply:
OK, didn't know about it.